recommend/admissionControllerPolicy.go

```diff
@@ -0,0 +1,108 @@
+package recommend
+
+import (
+	"context"
+	"errors"
+	"github.com/accuknox/auto-policy-discovery/src/libs"
+	"github.com/accuknox/auto-policy-discovery/src/protobuf/v1/worker"
+	"github.com/clarketm/json"
+	"github.com/fatih/color"
+	"github.com/kubearmor/kubearmor-client/k8s"
+	"github.com/kubearmor/kubearmor-client/utils"
+	kyvernov1 "github.com/kyverno/kyverno/api/kyverno/v1"
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/exp/slices"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"os"
+	"strconv"
+	"strings"
+)
+
+var connection *grpc.ClientConn
+
+func initClientConnection(c *k8s.Client) error {
+	if connection != nil {
+		return nil
+	}
+	var err error
+	connection, err = getClientConnection(c)
+	if err != nil {
+		return err
+	}
+	log.Info("Connected to discovery engine")
+	return nil
+}
+
+func closeConnectionToDiscoveryEngine() {
+	if connection != nil {
+		err := connection.Close()
+		if err != nil {
+			log.Println("Error while closing connection")
+		} else {
+			log.Info("Connection to discovery engine closed successfully!")
+		}
+	}
+}
+
+func getClientConnection(c *k8s.Client) (*grpc.ClientConn, error) {
+	gRPC := ""
+	targetSvc := "discovery-engine"
```

Comment:
can we keep the svc name, port, and labels for the discovery engine as part of constants or have them configurable?